core/payment-driver/zksync/src/zksync/wallet.rs

```diff
@@ -259,6 +270,20 @@ fn get_rpc_addr_from_env(network: Network) -> Option<String> {
     }
 }
 
+fn get_ethereum_node_addr_from_env(network: Network) -> String {
+    match network {
+        Network::Mainnet => std::env::var("ZKSYNC_MAINNET_GETH_ADDR")
```

Comment:
`std::env` is already `use`'d at the top of the file

```suggestion
        Network::Mainnet => env::var("ZKSYNC_MAINNET_GETH_ADDR")
```